scripts/oracle/getPrices.js

```diff
@@ -3,18 +3,18 @@ const ChainlinkPairAggregator = artifacts.require("./providers/chainlink/Chainli
 
 // Util classes
 const assert = require('assert');
-const ProcessArgs = require('./utils/ProcessArgs');
+const ProcessArgs = require('../utils/ProcessArgs');
 const processArgs = new ProcessArgs();
 
-const tokenName = 'DAI'.toUpperCase();
+const tokenName = 'USDC'.toUpperCase();
```

Comment:
This is a script, so you could run it on any network. I am working on a way to pass the token name (and other params) using the command line when you execute the script. It will be added to a new PR (for most of the scripts).
For the integration tests, you are right. We should add new ones to tests both tokens. I will create a task.